### DISCUSSION TITLE
[Feature Request] highly customizable Bottom Sheet

### DISCUSSION THREAD

**Discussion by amirhd-com** [heroui-inc/heroui#4708] [Feature requests]
### Is your feature request related to a problem? Please describe.

you can see https://github.com/stipsan/react-spring-bottom-sheet

### Describe the solution you'd like

clone powerful Bottom Sheet that I shared as an example

### Describe alternatives you've considered

its so useful in modern PWAs in this component library

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1796/[feature-request]-highly-customizable-bottom-sheet">ENG-1796 [Feature Request] highly customizable Bottom Sheet</a></p>
